mod/item.php

```diff
@@ -38,47 +37,30 @@
 use Friendica\Core\Worker;
 use Friendica\Database\DBA;
 use Friendica\DI;
-use Friendica\Model\Attach;
 use Friendica\Model\Contact;
-use Friendica\Model\Conversation;
-use Friendica\Model\FileTag;
 use Friendica\Model\Item;
 use Friendica\Model\ItemURI;
-use Friendica\Model\Notification;
 use Friendica\Model\Photo;
 use Friendica\Model\Post;
-use Friendica\Model\Tag;
-use Friendica\Model\User;
 use Friendica\Network\HTTPException;
-use Friendica\Object\EMail\ItemCCEMail;
 use Friendica\Protocol\Activity;
-use Friendica\Security\Security;
 use Friendica\Util\DateTimeFormat;
-use Friendica\Util\ParseUrl;
 
 function item_post(App $a) {
-	if (!DI::userSession()->isAuthenticated()) {
+	$uid = DI::userSession()->getLocalUserId();
+
+	if (!DI::userSession()->isAuthenticated() || !$uid) {
```

Comment:
You only need the $uid part as the “isAuthenticated” method will alway be true if the $uid is set.
```suggestion
	if (!$uid) {
```

Reply:
Done